tests/ui/doc/missing_ticks.rs

```diff
@@ -0,0 +1,9 @@
+// run-rustfix
```

Comment:
The usage of `// run-rustfix` is correct. However, could you add the comment to the `tests/ui/doc/doc.rs` file? It's usually better to have all test cases for a lint in one file, and I would like the suggestion to work for each suggestion and not just one. The rest looks good to me. :upside_down_face:

Reply:
I think this test file can be removed again now that `tests/ui/doc/doc.rs` has the fix comment and has more in depth tests